package-lock.json

```diff
@@ -1,251 +1,99 @@
 {
   "name": "synergismofficial",
   "version": "2.5.0-alpha",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Would you mind updating your Nodejs version to >=15.0.0 so the package.json structure is updated to v2? Thanks!

Reply:
I quite strongly prefer the LTS version of node / npm, as I use it in my day job.
nvm on windows has its issues, so I would prefer not having to update.
Do you use any of the feature node latest has over the LTS?

Reply:
I nevertheless installed it once to spare you some work.